test/clj/puppetlabs/trapperkeeper/services/webserver/normalized_uri_helpers_test.clj

```diff
@@ -116,8 +116,8 @@
     ;;> or the surrogate pair ED A1 8C ED BE B4 into U+233B4.  Decoding
     ;;> invalid sequences may have security consequences or cause other
     ;;> problems.
-    (is (= "��" (normalize-uri-path-for-string "%C0%AE")))
-    (is (= "/foo/��/��" (normalize-uri-path-for-string "/foo/%C0%AE/%C0%AE")))))
+    (is (= "-64-82" (normalize-uri-path-for-string "%C0%AE")))
```

Comment:
ok, seems a little odd, but I guess that makes some sense.  Would be good to update the comments in this tests and the `normalize-uri-path` docs in the README.md to cover what happens in this case.  Ultimately doesn't seem like a security vulnerability anyway for us since we always put the middleware which uses this normalization in front of our tk-auth middleware so doesn't seem too bad.